### PR TITLE
Create InEtn210320.xml

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: StPetersburg/InEtn210320.xml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0"
+     xml:lang="en"
+     xml:id="InEtn210320"
+     type="mss">
+  <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Prayer to Jesus Christ, Doctrine of Mysteries</title>
```
Comment (thea-m) [inline StPetersburg/InEtn210320.xml]: I believe that we once decided to give as title the transcribed titles of the main work, whenever possible (so for təmhərta ḫəbuʾāt only here), but there is a lot of general inconsistency regarding this in all our records, and I do not find this in the guidelines now. I will open an issue.

File: StPetersburg/InEtn210320.xml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                     <idno>Platonov cat. III.107</idno>
+                  </altIdentifier>
+               </msIdentifier>
+               <msContents>
+                  <summary/>
+                  <msItem xml:id="ms_i1">
+                    <locus from="1v" to="5r"/>
+                     <title type="incomplete" ref="LIT5831PrayerJesus"/>
+                     <textLang mainLang="gez"/>
+                    <incipit xml:lang="gez">ይዌድስ፡ ወቦቱ፡ ኵሉ፡ ማኅበረ፡ ፍትሱማን፡</incipit>
```
Comment (thea-m) [inline StPetersburg/InEtn210320.xml]: If it is ፍትሱማን፡ and not ፍጹማን in the catalogue/manuscript, maybe mark up with sic?

File: StPetersburg/InEtn210320.xml [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+                    <note>The beginning of the text is missing.</note>
+                  </msItem>
+                  <msItem xml:id="ms_i2">
+                    <locus from="7r" to="21r"/>
+                    <title type="incomplete" ref="LIT2444Temher"></title>
```
Comment (thea-m) [inline StPetersburg/InEtn210320.xml]: do you have any information in the catalogue regarding the kind and place of incompleteness?

File: StPetersburg/InEtn210320.xml [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+                  </objectDesc>
+                  <handDesc>
+                     <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script"/>
+                        <seg type="ink">Black, red</seg>
+                        <note>According to
+                          <bibl>
+                          <ptr target="bm:Chernetsov1975Svitki"/>
+                          <citedRange unit="page">215</citedRange>
+                        </bibl> the script is small and inacurate.</note>
```
Comment (thea-m) [inline StPetersburg/InEtn210320.xml]: inaccurate

File: StPetersburg/InEtn210320.xml (hunk 3 above)
Comment (DariaElagina) [reply StPetersburg/InEtn210320.xml]: done

File: StPetersburg/InEtn210320.xml (hunk 4 above)
Comment (DariaElagina) [reply StPetersburg/InEtn210320.xml]: done

File: StPetersburg/InEtn210320.xml (hunk 1 above)
Comment (DariaElagina) [reply StPetersburg/InEtn210320.xml]: done

File: StPetersburg/InEtn210320.xml (hunk 2 above)
Comment (DariaElagina) [reply StPetersburg/InEtn210320.xml]: that is a mistake, I have a new keyman :)

**Pull request merged**